library/core/src/fmt/num.rs

```diff
@@ -638,21 +639,29 @@ fn parse_u64_into<const N: usize>(mut n: u64, buf: &mut [MaybeUninit<u8>; N], cu
 #[stable(feature = "rust1", since = "1.0.0")]
 impl fmt::Display for u128 {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        fmt_u128(*self, true, f)
+        if cfg!(no_iu128_fmt) {
+            panic!("u128 formatting support is turned off")
```

Comment:
Is it possible to disable the Display impl entirely instead?

Reply:
I've tried a bit but removing it entirely would be a lot more intrusive since it is used in some other parts of `core`.

I might be able to remove it, the current implementation is mostly a POC for a quick test.